src/pages/Homepage/HeroSlideshow.vue

```diff
@@ -1,182 +1,138 @@
-			</kv-carousel-slide>
-			<kv-carousel-slide>
-				<kv-responsive-image v-if="counter > 3" :images="heroImages(4)" />
-			</kv-carousel-slide>
-			<kv-carousel-slide>
-				<kv-responsive-image v-if="counter > 4" :images="heroImages(5)" />
-			</kv-carousel-slide>
-		</kv-carousel>
-		<div class="headline-bar">
-			<div class="mask">
-				&nbsp;
-			</div>
-			<!-- -------------
-				Uncomment when the Billion To Women Campaign ends on Oct. 13th 2019
-			-------------------- -->
-
-			<!-- <div class="headline" v-if="showMgPromo">
-				<span class="headline-title">Make a bigger impact.</span>
-				<p class="headline-body">
-					A Monthly Good subscription<br class="su">
-					makes supporting entrepreneurs<br class="su">
-					part of your routine with monthly contributions.
-				</p>
-			</div> -->
-
-			<div class="headline billion-to-women-headline" v-if="promoEnabled">
-				<span class="headline-title">We just hit $1 billion in loans to women.</span>
-				<p class="headline-body">
-					To celebrate, we'll give you a free<br class="su">
-					$25 credit after you make a loan this week.
-				</p>
-			</div>
-			<div class="headline" v-else>
-				<span class="headline-title">Dreams are universal,<br class="smo"> opportunity is not.</span>
-				<p class="headline-body">
-					Lend as little as $25<br class="so">
-					to create<br class="mo">
-					opportunity<br class="so"><br class="lu">
-					for people<br class="mo">
-					around the world.
-				</p>
-			</div>
-			<div class="mask">
-				&nbsp;
-			</div>
-		</div>
-		<div class="action-button-wrapper">
-			<!-- -------------
-			Uncomment when the Billion To Women Campaign ends on Oct. 13th 2019
-			-------------------- -->
-
-			<!-- <kv-button
-				v-if="showMgPromo"
-				class="slideshow-action-button"
-				to="/monthlygood"
-				v-kv-track-event="['Home', 'EXP-Montly-Good-Promo', 'click-Sign-up']"
-			>
-				Sign up
-				<span v-if="showDoubleArrowButton"> >></span>
-			</kv-button> -->
-
-			<kv-button
-				v-if="promoEnabled"
-				class="slideshow-action-button billion-to-women-action"
-				to="/lend/lendingReward?lending_reward=265&doneUrl=lend-by-category/women"
-				v-kv-track-event="['Home', 'Lend now', 'click-Promo 1 Billion to Women 2019']"
-			>
-				Lend now
-				<span v-if="showDoubleArrowButton"> »</span>
-			</kv-button>
-			<kv-button
-				v-else
-				class="slideshow-action-button"
-				to="/lend-by-category"
-				v-kv-track-event="['Home', 'EXP-HeroWhyKiva', 'click-Start-lending', null, 'true']"
-			>
-				Start lending
-				<span v-if="showDoubleArrowButton"> »</span>
-			</kv-button>
-		</div>
+	<div>
+		<!-- Promo -->
+		<kv-hero v-if="promoEnabled" headline-bg-color="#00244e" class="possibilities-promo">
+			<template v-slot:images>
+				<kv-carousel @change="slideChange">
+					<kv-carousel-slide>
+						<kv-responsive-image
+							:images="possibilitiesImages(1)"
+							alt="An African American woman in a park wearing a leather jacket, smiling broadly"
```

Comment:
@mcstover Is there someone I can run wording like this past?

Reply:
Your best copy will work fine for these right now. There is no process (or time in this case) to go back through and do these.

In the future as we do these image heavy hero spaces we should start to ask for specific alt text against each image otherwise we'll just make it up according to what we see (this is what I usually do ;).